apps/src/aiTutor/levelEditors/AITutorFields.tsx

```diff
@@ -0,0 +1,75 @@
+import React, {useState} from 'react';
+import moduleStyles from '@cdo/apps/lab2/levelEditors/aichatSettings/edit-aichat-settings.module.scss';
+import Checkbox from '@cdo/apps/componentLibrary/checkbox/Checkbox';
+import {generalChatSystemPrompt} from '../constants';
+
+interface AITutorFieldsProps {
+  isAvailable: boolean;
+  levelbuilderPrompt: string;
+}
+
+const AITutorFields: React.FunctionComponent<AITutorFieldsProps> = ({
+  isAvailable,
+  levelbuilderPrompt,
+}) => {
+  const [aiTutorAvailable, setAiTutorAvailable] =
+    useState<boolean>(isAvailable);
+  const [levelSpecificPrompt, setlevelSpecificPrompt] =
+    useState<string>(levelbuilderPrompt);
+
+  return (
+    <div>
+      <input
+        type="hidden"
+        id="level_ai_tutor_available"
+        name="level[ai_tutor_available]"
+        value={aiTutorAvailable.toString()}
+      />
+
+      <div className={moduleStyles.fieldSection}>
+        <div className={moduleStyles.fieldRow}>
+          <label
+            htmlFor="aiTutorAvailableCheckbox"
+            className={moduleStyles.inlineLabel}
+          >
+            Enable AI Tutor on Level
+          </label>
+          <Checkbox
+            name="aiTutorAvailableCheckbox"
+            checked={aiTutorAvailable || false}
+            onChange={e => {
+              setAiTutorAvailable(!aiTutorAvailable);
+            }}
+          />
+        </div>
+        <br />
+
+        <input
+          type="hidden"
+          id="level_ai_tutor_level_specific_prompt"
+          name="level[ai_tutor_level_specific_prompt]"
+          value={levelSpecificPrompt.toString()}
+        />
+        {aiTutorAvailable && (
+          <div className={moduleStyles.fieldRow}>
+            <div className={moduleStyles['field-value']}>
+              <label htmlFor="levelPrompt">Level-Specific Prompt</label>
+              <textarea
+                id="levelPrompt"
+                value={levelSpecificPrompt || ''}
+                className={moduleStyles.textarea}
+                placeholder={`Enter a prompt specific to this level to be added with the default AI Tutor system prompt:\n${generalChatSystemPrompt}`}
```

Comment:
would it be more helpful to have this information above the input box so users can see the general prompt while they are writing, rather than before they start?

Reply:
@molly-moen What do you think of this? 
![image](https://github.com/code-dot-org/code-dot-org/assets/95503833/fb12740c-a54b-4c7a-a464-bd1bf196c5e5)

Reply:
that looks good!